hrms/overrides/employee_master.py

```diff
@@ -65,6 +65,16 @@ def update_employee_transfer(doc, method=None):
 		emp_transfer.db_set("new_employee_id", "")
 
 
+def update_job_applicant_and_offer(doc, method=None):
+	"""Updates Job Applicant and Job Offer status as 'Accepted' and submits them"""
+	if doc.job_applicant:
+		frappe.db.set_value("Job Applicant", doc.job_applicant, "status", "Accepted")
+		job_offer = frappe.get_last_doc("Job Offer", filters={"job_applicant": doc.job_applicant})
+		job_offer.status = "Accepted"
```

Comment:
```suggestion
		job_offer.status = "Accepted"
		job_offer.ignore_mandatory = True
		job_offer.ignore_permissions = True
```

If someone has customized the Job Offer doctype to make some field mandatory or doesn't have permission to update it, this will error out. Since its a backend operation, user will get confused in case of permission/mandatory errors from another doctype, so lets do it with these flags

Reply:
Also inform the user about this implicit action with a msgprint